### ISSUE TITLE
Keep versioned files build from multiple tasks

### ISSUE THREAD

**Issue by Mevrael** [laravel/elixir#456]
Is it possible to keep versioned files when use .version([assets]) and which was not passed to version() method but exists in rev-manifest.json?

I have tasks to build assets for admin panel, app itself and some minor tasks like svg icon versioning, however right now I have to run single .version() and pass all app/admin/svg assets to it because it rebuilds rev-manifst from the scratch and removes everything not passed to it.

In other words:
1) with `gulp task1` you have mix.version('task1.file');
2) with `gulp task2` you have mix.version('task2.file');

However if you run task2 your task1.file is removed from manifest and vice versa.

**Comment by JeffreyWay** [comment]
Hmm, yeah right now you should stick with one call to `mix.version()`.
